gocourse02/main.go

```diff
@@ -0,0 +1,136 @@
+package main
+
+import "fmt"
+
+// Struct that represents an Animal
+type Animal struct {
```

Comment:
В загальному, я не прихильник коментарів, де й так дуже зрозумілий код. Але хай вже будуть.

Якщо вже лишати коментарі до структур та полів, то краще робити це по Go-шному. Ось тут детально розписано https://go.dev/doc/comment

Суть в тому, що перше слово має збігатись з назвою структури чи функції. Також наприкінці речення обов'язкова крапка:
```suggestion
// Animal represents an animal in a zoo.
type Animal struct {
```

Reply:
Removed all comments